portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiMain.java

```diff
@@ -41,16 +58,119 @@ public void uncaughtException(Thread thread, Throwable t) {
           }
         });
 
+    ApiMain apiMain = new ApiMain();
+    apiMain.initialize();
+    apiMain.start();
+  }
+
+  public void initialize() {
+    TypeManager typeManager = new TypeManagerImpl();
+
+    // TODO implement
+    Map<String, Object> configuration = new HashMap<>();
```

Comment:
FYI feel free to use the config parsing stuff from WorkerExtensionContext I added, if it'd be helpful

String tempSettings = "environment: LOCAL\ncloud: LOCAL";
InputStream in = new ByteArrayInputStream(tempSettings.getBytes(StandardCharsets.UTF_8));
config = ConfigUtils.parse(in);